nixos/modules/installer/cd-dvd/iso-image.nix

```diff
@@ -433,7 +445,11 @@ let
   '';
 
   efiImg = pkgs.runCommand "efi-image_eltorito" {
-    nativeBuildInputs = [ pkgs.buildPackages.mtools pkgs.buildPackages.libfaketime pkgs.buildPackages.dosfstools ];
```

Comment:
This is going to be automatically formatted at some point anyway, no need for unnecessary noise